README.md

```diff
@@ -68,14 +74,14 @@ For official benchmarking:
 * **Latency**: The total time each request spent in the IO path. *Lower is better.*
     * It's a measurement of how efficient is the storage system to handle each request.
     * The data path overhead of a storage system can be expressed as the latency it added on top of the native storage system (SSD/NVMe).
-* **CPU Idleness**: How idle is the CPU on the node that's running the test. *Higher is better.*
+* **CPU Utilization**: How busy is the CPU spent in kernel (sys) and in user-mode codes (usr) on the node that's running the test. *Lower is better.*
```

Comment:
It's not `lower is better`.

https://fio.readthedocs.io/en/latest/fio_doc.html#interpreting-the-output
```
cpu
CPU usage. User and system time, along with the number of context switches this thread went through, usage of system and user time, and finally the number of major and minor page faults. The CPU utilization numbers are averages for the jobs in that reporting group, while the context and fault counters are summed.
```

So if the CPU is lower, it only means less CPU used by FIO, which can be caused by:
1. Other processes took more CPU cycles so there isn't much left for FIO.
2. The other processes cannot use more CPU before hitting their limit so FIO can use more.

I used idleness profile (https://fio.readthedocs.io/en/latest/fio_doc.html#cpu-idleness-profiling) before it's basically taking all the remaining CPU with a lowest priority task, which means those are the CPU that can be used in the environment. So we can say for more if it's higher, the CPU is more `idle`.

Reply:
> It's not `lower is better`.
> 
> https://fio.readthedocs.io/en/latest/fio_doc.html#interpreting-the-output
> 
> > cpu
> > CPU usage. User and system time, along with the number of context switches this thread went through, usage of system and user time, and finally the number of major and minor page faults. The CPU utilization numbers are averages for the jobs in that reporting group, while the context and fault counters are summed.
> 
> So if the CPU is lower, it only means less CPU used by FIO, which can be caused by:
> 
> 1. Other processes took more CPU cycles so there isn't much left for FIO.
> 2. The other processes cannot use more CPU before hitting their limit so FIO can use more.
Right. The statement, "lower is better", is true when there is no other noise from other applications. 
> 
> I used idleness profile (https://fio.readthedocs.io/en/latest/fio_doc.html#cpu-idleness-profiling) before it's basically taking all the remaining CPU with a lowest priority task, which means those are the CPU that can be used in the environment. So we can say for more if it's higher, the CPU is more `idle`.